policies/module-types/commands/rudder_module_type.yml

```diff
@@ -0,0 +1,118 @@
+---
+name: Commands
+description: Execute commands
+version: "0.0.1"
+supported_targets: ["unix"]
+parameters:
+  command:
+    description: "Command to be executed"
+    constraints:
+      allow_empty: false
+      allow_whitespace_string: false
+    p_type: String
+  args:
+    description: "Arguments to the command"
```

Comment:
Here the args is listed as a String.
But in the module doc we have:
`| `args` | Arguments to the command | A list of strings or empty (if `in_shell` is set to `true`) | No |`

This is not clear enough. Also, the tests are missing arguments heavy examples.